builtin/files/cluster.yaml.tmpl

```diff
@@ -1326,6 +1326,15 @@ kubernetesDashboard:
 kubeDns:
   # Define which DNS provider to use (kube-dns or coredns), default coredns.
   provider: coredns
+  
```

Comment:
Why not just make this resources and make it work for either kubedns or coredns?

Reply:
Done!

Reply:
Awsome! :)